indico/modules/events/registration/templates/management/registration_modify.html

```diff
@@ -7,6 +7,7 @@
 {% endblock %}
 
 {% block content %}
+    {{ template_hook('before-regform', event=event, regform=regform, management=management, registration=none) }}
```

Comment:
I don't think this one is correct. `registration=none` but we are modifying an existing registration here so we should pass it..

Reply:
Also, I'd add an `after-regform` to be consistent

Reply:
You're right it's not correct indeed.
Thank you to noticed it.
I'm going to fix it